ui/plugins/annotations/controller.js

```diff
@@ -42,7 +38,9 @@ treeherder.controller('AnnotationsPluginCtrl', [
             var jobMap = ThResultSetStore.getJobMap($rootScope.repoName);
             var job = jobMap[key].job_obj;
 
-            job.failure_classification_id = 1;
+            // this $evalAsync will be sure that the * is added or removed in
+            // the job in the jobs view area when this change takes place.
```

Comment:
I'd maybe say instead of the "jobs view area", perhaps say ''....in the jobs table" here.